File: ClinicAPI/ViewModels/ServiceForm/XqFormModel.cs
using DAL.Core;
using System;
namespace ClinicAPI.ViewModels.ServiceForm
{
    public class XqFormModel
    {
        public string IdCode { get; set; }
        // Chẩn đoán
        public string DiagnosisName { get; set; }
        // Ngày kê đơn
        public DateTime DateCreated { get; set; }
        public PrescriptionStatus Status { get; set; }

        // Phiếu chụp X Quang
        // Yêu cầu
        public string Request { get; set; }
        // Ghi chú
        public string Note { get; set; }
    }

    public class XqFormAddModel : XqFormModel
    {
        public int PatientId { get; set; }
        public int HistoryId { get; set; }
    }

    public class XqFormViewModel : XqFormModel
    {
        public int Id { get; set; }

        public int PatientId { get; set; }
        public PatientPartialViewModel Patient { get; set; }
        public string DoctorId { get; set; }
        public DoctorViewModel Doctor { get; set; }
    }

    public class XqFormUpdateModel : XqFormModel
    {
    }
}

File: ClinicApp/reactapp/src/models/XqFormModel.js
import {
    PrescriptionStatusEnum,
    PrescriptionStatus,
} from "../constants";
import moment from 'moment';

export const XqFormModel = {
    IdCode: '',
    // Chẩn đoán
    DiagnosisName: '',
    // Ngày kê đơn
    DateCreated: moment(),
    Status: PrescriptionStatusEnum[PrescriptionStatus.IsNew],

    // Phiếu chụp X Quang
    // Yêu cầu
    Request: '',
    // Ghi chú
    Note: '',
};
